apps/data/priv/repo/migrations/20191004151514_update_games_table.exs

```diff
@@ -0,0 +1,11 @@
+defmodule GrapevineData.Repo.Migrations.UpdateGamesTable do
+  use Ecto.Migration
+
+  def up do
+    execute "UPDATE games SET description = '' WHERE description IS NULL"
```

Comment:
Do we want to set description `null: false` with this? There are <10 events on production so I'm not worried about that migration 😄

Reply:
👍